awx/sso/pipeline.py

```diff
@@ -77,6 +77,15 @@ def _update_m2m_from_expression(user, related, expr, remove=True):
         related.remove(user)
 
 
+def get_or_create_with_default_galaxy_cred(**kwargs):
+    from awx.main.models import Organization
+
+    (org, org_created) = Organization.objects.get_or_create(kwargs)
```

Comment:
I mentioned this in our side-bar, but this probably needs to be `**kwargs`, and I believe that could be why unit tests are failing. I'm happy to approve this once unit tests are passing, because this is a solid fix.